src/openrct2-ui/windows/RideList.cpp

```diff
@@ -172,872 +143,795 @@ static constexpr const rct_string_id page_names[] = {
+                        return ride_customers_per_hour(thisRide) <= ride_customers_per_hour(otherRide);
+                    });
+                    break;
+                case INFORMATION_TYPE_AGE:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->build_date <= otherRide->build_date;
+                    });
+                    break;
+                case INFORMATION_TYPE_INCOME:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->income_per_hour <= otherRide->income_per_hour;
+                    });
+                    break;
+                case INFORMATION_TYPE_RUNNING_COST:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->upkeep_cost <= otherRide->upkeep_cost;
+                    });
+                    break;
+                case INFORMATION_TYPE_QUEUE_LENGTH:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->GetTotalQueueLength() <= otherRide->GetTotalQueueLength();
+                    });
+                    break;
+                case INFORMATION_TYPE_QUEUE_TIME:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->GetMaxQueueTime() <= otherRide->GetMaxQueueTime();
+                    });
+                    break;
+                case INFORMATION_TYPE_RELIABILITY:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->reliability_percentage <= otherRide->reliability_percentage;
+                    });
+                    break;
+                case INFORMATION_TYPE_DOWN_TIME:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->downtime <= otherRide->downtime;
+                    });
+                    break;
+                case INFORMATION_TYPE_GUESTS_FAVOURITE:
+                    SortList(currentListPosition, ridePtr, [](const Ride* thisRide, const Ride* otherRide) -> bool {
+                        return thisRide->guests_favourite <= otherRide->guests_favourite;
+                    });
+                    break;
+            }
+            listIndex++;
+        }
+
+        this->selected_list_item = -1;
+        this->Invalidate();
+    }
+
+    // window_ride_list_close_all
+    void CloseAllRides()
+    {
+        for (auto& rideRef : GetRideManager())
```

Comment:
```suggestion
        for (auto& ride : GetRideManager())
```

Reply:
Can't be called just "ride" since rct_window already has a member called "ride".

Reply:
I see, ignore that then, sorry about that.